pyhyp/pyHyp.py

```diff
@@ -555,7 +410,7 @@ def __init__(self, comm=None, options=None, debug=False):
         # Convert file type to integer
         fileType = {"cgns": self.hyp.hypinput.cgnsfiletype, "plot3d": self.hyp.hypinput.plot3dfiletype}
 
-        intFileType = fileType[self._go("fileType").lower()]
+        intFileType = fileType[self._go("fileType")]
```

Comment:
Only thing from me is whether we want to throw away the `self._go` function and instead just say `self.getOption()`, which is far more explicit and the extra characters are probably not a big deal.

Reply:
I agree that using `self.getOption()` is better. I'll change that.